### DISCUSSION TITLE
ZIP code with multiple areas

### DISCUSSION THREAD

**Discussion by moonwalker2004** [Second-Hand-Friends/kleinanzeigen-bot#167] [Q&A] [answered]
Hi people!

First of all, thanks for this wonderful tool - this will greatly help me with advertising my CD and DVD collection!

I have a question about the ZIP code configuration in "config.yaml". As far as I can see, it's only possible to put a ZIP code but not specify the area. Some ZIP codes have multiple areas attached such as the ZIP code 60327:

PLZ 60327 --> Frankfurt am Main Gallusviertel
PLZ 60327 --> Frankfurt am Main Westend
PLZ 60327 --> Frankfurt am Main Gutleutviertel

When looking for local ads, I quite often search in the local area/ Viertel/ Kiez. With the "zipcode" setting in "config.yaml", it seems that it is always the first area that is selected. Is there a way to specify the area in the config or it that something that could be added as a feature?

Best wishes,
Markus

EDIT: When removing the "zipcode" line, it seems that last entry will be used so this solves my issue. However, when posting with different locations, the above could be relevant.

**Comment by 1cu** [comment, marked as answer]
This has been solved in #384.

Use location in your yaml to set the correct district.
